it/src/main/java/org/corfudb/universe/scenario/fixture/Fixtures.java

```diff
@@ -0,0 +1,139 @@
+package org.corfudb.universe.scenario.fixture;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import lombok.Builder;
+import lombok.Getter;
+import org.corfudb.universe.service.Service.ServiceParams;
+import org.slf4j.event.Level;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+import static lombok.Builder.Default;
+import static org.corfudb.universe.cluster.Cluster.ClusterParams;
+import static org.corfudb.universe.node.CorfuServer.Mode;
+import static org.corfudb.universe.node.CorfuServer.Persistence;
+import static org.corfudb.universe.node.CorfuServer.ServerParams;
+
+/**
+ * Fixture factory provides predefined fixtures
+ */
+public interface Fixtures {
+
+    @Builder
+    @Getter
+    class SingleServerFixture implements Fixture<ServerParams> {
```

Comment:
Maybe we should be more descriptive this is a Fixture for a memory based single server fixture..